### PR TITLE
Fix Haddock failures on GHC 8.10

### PR DESCRIPTION
**Pull request by lag47**
Addressing haddock build errors described in the following issue.
https://github.com/GaloisInc/saw-script/issues/1719

### CODE REVIEW

File: src/SAWScript/Crucible/LLVM/Override.hs [hunk 1]
```diff
@@ -5,27 +5,26 @@ License     : BSD3
...
 Stability   : provisional
 -}
-{-# LANGUAGE DataKinds #-}
-{-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE GADTs #-}
+{-# LANGUAGE DataKinds                  #-}
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE FlexibleContexts           #-}
+{-# LANGUAGE GADTs                      #-}
```
Comment (RyanGlScott) [inline src/SAWScript/Crucible/LLVM/Override.hs:11]: This PR contains a fair bit of whitespace reorganization that, while likely useful, appears to be unrelated to the goal of fixing Haddock errors. For the sake of making the `git` history neater, it would be more convenient if these sorts of changes could be put in separate commits.

**Review by RyanGlScott — COMMENTED** [review]
Thanks, @lag47! This generally looks good to me, although I've left one inline request.

**Comment by sauclovian-g** [comment]
This never got merged because of the whitespace concerns; since we no longer support GHC 8 and don't have to satisfy its Haddock, I think at this point it should probably just be closed.
